documentation.html

```diff
@@ -56,11 +56,109 @@
+                                style="width: 677px; height: 204px; border:0; transform: scale(1); overflow:hidden;"
+                                sandbox="allow-scripts allow-same-origin">
+                            </iframe>
+                        </div>
+                    </div>
+                    <div class="mg-t-40">
+                        <h2 class="rg-booky-title">Buttons with Icons and Icon Buttons</h2>
+                        <div class="mg-t-30 btn-container">
+                            <div class="btn-examples">
+                                <button class="btn primary-btn solid icon-left">
+                                    <i class="fas fa-shopping-cart"></i>
+                                    Add to Bag
+                                </button>
+                                <button class="btn secondary-outlined-btn icon-right">
+                                    Add To Library
+                                    <i class="far fa-bookmark"></i>
+                                </button>
+                                <button class="btn icon-btn danger-icon">
+                                    <i class="fas fa-trash-alt"></i>
+                                </button>
+                                <button class="btn icon-btn">
+                                    <i class="fas fa-shopping-cart"></i>
+                                </button>
+                                <button class="btn icon-btn primary-icon">
+                                    <i class="fas fa-bookmark"></i>
+                                </button>
 
+                            </div>
+                        </div>
+                        <div class="code-example">
+                            <iframe class="code-snip"
+                                src="https://carbon.now.sh/embed?bg=rgba%28189%2C213%2C234%2C0%29&t=vscode&wt=sharp&l=auto&width=859&ds=false&dsyoff=20px&dsblur=68px&wc=true&wa=true&pv=0px&ph=0px&ln=false&fl=1&fm=Hack&fs=18px&lh=144%25&si=false&es=2x&wm=false&code=%253Cbutton%2520class%253D%2522btn%2520primary-btn%2522%253EPrimary%2520Button%253C%252Fbutton%253E%250A%253Cbutton%2520class%253D%2522btn%2520secondary-btn%2522%253ESecondary%2520Button%253C%252Fbutton%253E%250A%253Cbutton%2520class%253D%2522btn%2520success-btn%2522%253ESuccess%2520Button%253C%252Fbutton%253E%2520%250A%253Cbutton%2520class%253D%2522btn%2520warning-btn%2522%253EWarning%2520Button%253C%252Fbutton%253E%250A%253Cbutton%2520class%253D%2522btn%2520danger-btn%2522%253EError%2520Button%253C%252Fbutton%253E"
+                                style="width: 677px; height: 204px; border:0; transform: scale(1); overflow:hidden;"
```

Comment:
You can move this inline styling of iframe to a common class.

Reply:
They have small differences between each code example. does it mean I have to create a class for each code snippet?